MyApplication/app/src/main/java/com/courses/bogdan/myapplication/activities/MainActivity.java

```diff
@@ -79,22 +76,23 @@ public void onClick(View view) {
     @Override
     protected void onResume() {
         super.onResume();
-        Random randomColor = new Random();
-        red= randomColor.nextInt(256);
-        green = randomColor.nextInt(256);
-        blue = randomColor.nextInt(256);
-        layout.setBackgroundColor(Color.argb(255,red,green,blue));
+        setRandomColor();
         getButton.setEnabled(false);
     }
 
     @Override
     protected void onStart() {
         super.onStart();
+        setRandomColor();
+        getButton.setEnabled(true);
+
+    }
+
+    public void setRandomColor(){
         Random randomColor = new Random();
         red= randomColor.nextInt(256);
```

Comment:
Te rog extrage valoarea 256 intr-o constanta. Una din standardele Java este sa extragem astfel de valori in constante. Exceptile sunt valorile boolean (true si false) si valoarea 0.